vendor-extra/ContentPageBundle/src/DependencyInjection/ContentPageExtension.php

```diff
@@ -47,6 +47,7 @@ private function addPage(string $name, array $config, ContainerBuilder $containe
         $definition->setArgument(2, new Reference('twig'));
         $definition->setArgument(3, $config['page_template']);
         $definition->addTag('controller.service_arguments');
+        $definition->addTag('libero.content_page.controller', ['handler' => $config['handler']]);
```

Comment:
`handler` is overloaded, we'll find a metaphor for the naming I suppose, given publishing wealth of concepts and strange words